source/templates/displaying-a-list-of-items.md

```diff
@@ -21,15 +36,19 @@ The above example will print a list like this:
 </ul>
 ```
 
-The `{{#each}}` helper is bindings-aware.  If your
-application adds a new item to the array, or removes an item, the DOM
-will be updated without having to write any code. Note that a `[].push()`
-will not update the helper. Adding items need to be done with `[].pushObject`,
-and related [Ember Mutable Array methods](http://emberjs.com/api/classes/Ember.MutableArray.html) so that Ember can observe the change.
+Like other helpers, the `{{#each}}` helper is bound.  If a new item is added to
+or removed from the iterated array, the DOM will be updated without having to
+write any additional code.
 
-### Accessing the list item's `index`
+Ember requires that you use special methods to update bound arrays, for example
```

Comment:
It seems worth re-iterating? It is a common stumbling block for new users, and I fear not everyone will read the guides front-to-back.